cmd/agent/subcommands/integrations/command.go

```diff
@@ -42,19 +42,12 @@ const (
 	downloaderModule    = "datadog_checks.downloader"
 	disclaimer          = "For your security, only use this to install wheels containing an Agent integration " +
 		"and coming from a known source. The Agent cannot perform any verification on local wheels."
-	integrationVersionScriptPy2 = `
+	integrationVersionScript = `
 import pkg_resources
 try:
 	print(pkg_resources.get_distribution('%s').version)
 except pkg_resources.DistributionNotFound:
 	pass
-`
```

Comment:
I'm wondering if we should keep this part, because we want to get rid of setuptools eventually and this works perfectly fine. WDYT?

Reply:
Sounds good to me, if @DataDog/agent-shared-components is okay with this.